ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -1935,3 +1935,37 @@ spec:
   purpose: Informational
   tags: compliance, CIS, CIS_Level1, CIS6.3.6
   contributors: artemist-work
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure Show Full Website Address in Safari Is Enabled (FDA Required)
+  platforms: macOS
+  platform: darwin
+  description: |
+    Attackers use websites with malicious or unwanted content to exploit the user or the computer. Part of the attack chain is to lure someone to load their content rather than the desired content. In order to reduce the risk in interacting with unwanted content the full website address should always be displayed in Safari.
+  resolution: |
+    Graphical Method:
+    Perform the following steps to set Safari to show full website addresses:
+      1. Open Safari
+      2. Select Safari from the menu bar
+      3. Select Settings
+      4. Select Security
+      5. Set Show full website address to enabled
+    Automated method:
+    Profile Method:
+    Create or edit a configuration profile with the following information:
+      1. The PayloadType string is com.apple.Safari
+      2. The key to include is ShowFullURLInSmartSearchField 
+      3. The key must be set to:
+      <true/>
+  query: |
+    SELECT 1 WHERE NOT EXISTS (
+      SELECT * FROM plist
+      WHERE path LIKE '/Users/%/Library/Containers/com.apple.Safari/Data/Library/Preferences/com.apple.Safari.plist' 
+      AND key = 'ShowFullURLInSmartSearchField' 
+      AND value != 1
+    );
```

Comment:
This is similar to CIS check `6.1.1` and `6.3.6` where we have to check that there's an entry set to a specific value for ALL users. You should copy paste the query from those checks and amend the path, key and value.